includes/sanitizers/class-amp-base-sanitizer.php

```diff
@@ -501,7 +502,7 @@ public function remove_invalid_child( $node, $validation_error = [] ) {
 	 * @return bool Whether the node should have been removed, that is, that the node was sanitized for validity.
 	 */
 	public function remove_invalid_attribute( $element, $attribute, $validation_error = [], $attr_spec = [] ) {
-		if ( DevMode::isExemptFromValidation( $element ) ) {
+		if ( $this->is_exempt_from_validation( $element ) ) {
```

Comment:
I think this can be reverted based on my comment on the `sanitize` method below.